src/serial/SerialArea.tsx

```diff
@@ -19,9 +43,124 @@ const SerialArea = (props: BoxProps) => {
       position="relative"
       overflow="hidden"
     >
-      {!connected ? null : <XTerm flex="1 1 auto" height={0} />}
+      {!connected ? null : (
+        <Box
+          alignItems="stretch"
+          backgroundColor={backgroundColorTerm}
+          spacing={0}
+          height="100%"
+        >
+          <SerialBar
+            height={12}
+            compact={compact}
+            onSizeChange={onSizeChange}
+          />
+          <XTerm
+            visibility={compact ? "hidden" : undefined}
+            height="calc(100% - 40px)"
```

Comment:
This 40px is almost certainly wrong. Value should be shared between here and the split view.

Reply:
9265c8cdc1ae08b812e6f3e0debe2743fb0a503b